src/lib/ai/services/ComparativeAnalyticsFactory.ts

```
@@ -0,0 +1,156 @@
+      )
+    }
+
+    // Create or use provided benchmark repository
+    const benchmarkRepository =
+      options.benchmarkRepository || (await createDefaultBenchmarkRepository())
+
+    // Create redis instance if URL is provided
+    let redis = redisInstance
+    if (options.redisUrl) {
+      try {
+        const Redis = require('ioredis')
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)